financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkinglinksignup/NetworkingLinkSignupViewModel.kt

```diff
@@ -195,15 +197,23 @@ internal class NetworkingLinkSignupViewModel @AssistedInject constructor(
     /**
      * @param validEmail valid email, or null if entered email is invalid.
      */
-    private suspend fun onEmailEntered(
+    private fun onEmailEntered(
         validEmail: String?
     ) {
         setState { copy(validEmail = validEmail) }
         if (validEmail != null) {
             logger.debug("VALID EMAIL ADDRESS $validEmail.")
             searchJob += suspend {
                 delay(getLookupDelayMs(validEmail))
-                lookupAccount(validEmail)
+
+                val lookup = lookupAccount(validEmail)
+                val clientSecret = lookup.consumerSession?.clientSecret
+
+                if (lookup.exists && clientSecret != null) {
+                    startVerification.sms(consumerSessionClientSecret = clientSecret)
+                }
```

Comment:
@tillh-stripe what's the idea behind starting verification here? (likely [that can still be owned by the verification pane](https://github.com/stripe/stripe-android/pull/9032/files#diff-234fe86a77579b755abc47dd5e9a219813c625086280f1d9eb74e7efdd971b28R71)?)

Reply:
I don’t remember why I did this in the original draft. Can likely be removed.